app/assets/stylesheets/custom.css

```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
```

Comment:
- [ ] There are some overflow problems in the mobile view (tested both on Chrome and Firefox):

![image](https://github.com/roniy68/capstone-rails/assets/57421823/aa9c5a80-0fc2-4754-bd17-2d037408b285)

![image](https://github.com/roniy68/capstone-rails/assets/57421823/5d70fbb6-23a3-435d-b14c-e2e4fffa774e)

![image](https://github.com/roniy68/capstone-rails/assets/57421823/9f4db0ed-ca4e-4273-8e1e-f6334352d580)

Kindly fix these overflow problems on your styling, so your mobile app looks more professional and beautiful. 💪

Reply:
🔥 Fixed The Responsiveness

https://www.loom.com/share/ab17f762d1a44bad804d3d480ca30909?sid=33e79aad-b43b-4ec3-9537-0a73d4678dcf